Formula/blastem.rb

```diff
@@ -22,81 +22,42 @@ class Blastem < Formula
 
   depends_on "freetype" => :build
   depends_on "gettext" => :build
+  depends_on "imagemagick" => :build
   depends_on "jpeg" => :build
   depends_on "libpng" => :build
   depends_on "openjpeg" => :build
   depends_on "pkg-config" => :build
   depends_on "glew"
-  depends_on :macos # Due to Python 2
+  depends_on "pillow"
+  depends_on "python@3.9"
```

Comment:
We're shipping new (runtime) dependencies, so let's bump the `revision` here.

Reply:
Actually I made a mistake - both `python@3.10` and `pillow` are build time dependencies.  I think in this case we don't need a revision bump?